pocket_world_application/src/main/java/org/exschool/pocketworld/model/Production.java

```diff
@@ -0,0 +1,46 @@
+package org.exschool.pocketworld.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Production {
+	@ManyToOne
```

Comment:
you should use embedded id here http://stackoverflow.com/questions/3585034/how-to-map-a-composite-key-with-hibernate